scheduling-api-graphql/scheduling-api-graphql-fetchers/src/main/java/org/ow2/proactive/scheduling/api/graphql/fetchers/converter/JobInputConverter.java

```diff
@@ -177,6 +181,37 @@ public List<Predicate[]> inputToPredicates(DataFetchingEnvironment environment,
                 predicates.add(tenantPredicate);
             }
 
+            if (!Strings.isNullOrEmpty(cumulatedCoreTime)) {
```

Comment:
number values such as cumulatedCoreTime should not use a WildCardPredicate but rather a comparable predicate where you can filter with e.g. > value